src/pages/AllNewslettersPage.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react'
+import { Row, Col } from 'react-bootstrap'
+import styled from 'styled-components'
+import Header from '../components/Header'
+import Footer from '../sections/Footer'
+
+import { API_ROOT } from '../constants'
+
+
+const ReactMarkdown = require('react-markdown')
+const fs = require('fs')
+
+const Section = styled(Row)`
+    margin:50px;
+`
+
+
+class AllNewslettersPage extends Component {
+
+    state = {
+        markdownContent: null,
```

Comment:
This data no longer represents and "markdown content" so I think it'd make more sense semantically to rename to something like `newsletters` or `allNewsletters` or `allPublishedNewsletters`

Always good to make sure our variables and function names actually describe the underlying data and functionality